### ISSUE TITLE
Fill in Pass Scheduling Parameters

### ISSUE THREAD

**Issue by PandnotPthereforeQ** [oresat/uniclogs-software#25]
In addition to parameters already [here](https://github.com/oresat/uniclogs-software/blob/capstone-dev/cosmos/config/targets/ENGR_LINK/cmd_tlm/cmd.txt), we require the height of the target location, and an ID # for which camera or type of mission (OreSat Live, CFC, OIT, software update, etc). I think Loss of Signal might not be a necessary parameter, but it might be nice to not have the ADCS system have to figure out when to stop the mission.

In summary:

Schedule Pass:
 * int "Packet ID"
 * int "Pass ID"
 * int "Mission Mode ID"
 * str "Acquisition of Signal"
 * str "Loss of Signal" (optional)
 * float "Latitude"
 * float "Longitude"
 * float "Height"

**Comment by dmitri-mcguckin** [comment]
TODO first:
- [x] [Capstone Handoff](https://github.com/oresat/uniclogs-software/pull/24) merge into master

**Comment by gusbunce** [comment]
is this still needed to be done?
